src/Text/Pandoc/Writers/LaTeX.hs

```diff
@@ -1448,6 +1450,8 @@ stripLocatorBraces = walk go
   where go (Str xs) = Str $ T.filter (\c -> c /= '{' && c /= '}') xs
         go x        = x
 
+
```

Comment:
Small suggestion: it makes it easier to review patches if irrelevant changes like the addition of blank space here, or the inserted newline at 1471/2, are left out.